employee-onboarding-A2A-reference-implementation/agents/it_agent/agent.py

```
@@ -0,0 +1,183 @@
+                    await _broadcast(f"[IT_PROVISIONED] ✅ IT access provisioned for {employee_name} ({employee_id}): {result[:300]}")
+                elif final_status == "rejected":
+                    logger.warning(f"[IT_AGENT] ❌ Rejected by admin — no resources provisioned for {employee_id}.")
+                    await _broadcast(f"[IT_PROVISIONED] ❌ IT access request rejected for {employee_name} ({employee_id}). No resources were provisioned.")
+                else:
+                    logger.warning(f"[IT_AGENT] ⏰ Approval timed out for {employee_id}.")
+                    await _broadcast(f"[IT_PROVISIONED] ⏰ IT access approval timed out for {employee_name} ({employee_id}). Please retry.")
+            except Exception as e:
+                logger.error(f"[IT_AGENT] Background provisioning failed: {e}", exc_info=True)
+                await _broadcast(f"[IT_PROVISIONED] ❌ IT provisioning error for {employee_name} ({employee_id}): {e}")
+        import asyncio
+        asyncio.create_task(_wait_and_provision())
```

Ruff (0.15.2)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)